docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,97 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+## Before you begin
+* [Learn about unix epoch/unix time](https://en.wikipedia.org/wiki/Unix_time){:target="_blank"}
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function uses the Unix Epoch to calculate a date-time equivalent of a specified integer, then returns the results.
+
+## Syntax
+
+```
+TOTIMESTAMP([int_val | expr], timeunit)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| int_val | integer | Integer value to be converted to a timestamp | Yes | |
+| expr | | expression used to derive an integer value to be converted to a timestamp | Optional | |
+| timeunit | string | String value that specifies the unit of time to convert. | Optional | [Time units](#additional-information).|
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `timestamp` | Returns timestamp equivalent of `int_exp`, it converts the input using unix epoch as base and the optional `timeunit` to determine the granularity of the input value. |
+
+## Additional information
+
+### `timeunit`
+
+`timeunit` defaults to `s` if not specified.
+
+{% include /sql-guide/timestamp-timeunit-table.md %}
+
+## Examples
+
+### Implicitly convert integer to timestamp. 
+Implicit conversion will treat the integer value as seconds since unix epoch. Seconds is the default granularity of integer date-time value.
+
+```sql
+create table demo
+    (_id id, ts timestamp timeunit 's');
+
+insert into demo(_id, ts)
```

Comment:
Can these be just one insert into statement? this makes the example and workflow longer for users:

insert into demo(_id, ts) values (1, 0), (2, 86400), (3, 90061), (4, -86400);